libs/ledgerjs/packages/hw-app-eth/src/utils.ts

```diff
@@ -115,3 +115,28 @@ export const decodeTxInfo = (rawTx: Buffer) => {
  */
 export const intAsHexBytes = (int: number, bytes: number): string =>
   int.toString(16).padStart(2 * bytes, "0");
+
```

Comment:
Maybe a good place for a comment to explain what a selector is.

Reply:
Done 👌